### PR TITLE
DR7 FUSE updates

### PR DESCRIPTION
**Pull request by jotaylor**
Two updates:
- add in the new DR7 FUSE targets
- rename all targets in the FUSE yaml files to use the new aliases

### CODE REVIEW

File: utils/data/fuse/good_fuse_targs.yaml [hunk 1]
```diff
@@ -133,31 +130,43 @@ DR6:
...
+- AV-388
+- LH-114-7
+- AV-83
+- BI-272
+DR7:
+- NGC346-MPG-435
+- N11-ELS-032
+- BI-13
+- AV-506
+- BI-128
```
Comment (Rplesha) [inline utils/data/fuse/good_fuse_targs.yaml:161]: This target was dropped from the sample. It should be moved to the bad fuse list

**Review by Rplesha — COMMENTED** [review]
One more flagging situation:
- SK-69D106: ULLYSES-1618
  - Assessment:  Only LiF1A and LiF1B spectra are immediately useful.  SiC1B spectrum might be salvageable, but will have very low S/N.

**Comment by jotaylor** [comment]
@Rplesha How do you feel about merging this as is, then coming back to update target HD35517 once Alex has looked at it? (Also, I assume that is the target you meant before, not SK-69D106)

**Comment by Rplesha** [comment]
@jotaylor SK-69D106 = HD-35517! 

If you want to merge this to run the products tonight, then I'll approve it. Otherwise, I think it can wait until Alex is able to get back to us.

**Review by Rplesha — APPROVED** [review]

**Comment by Rplesha** [comment]
From Alex: Only the LiF1A and LiF1B spectra are useful [for HD35517], so the "active" wavelength range will be from 992 - 1188 Angstroms.   The gap between the detector segment between 1079.9 and 1094.1 Angstroms should be DQ flagged for exclusion from the HLSP.

**Review by Rplesha — APPROVED** [review]
Good from my end if you agree with my changes @jotaylor

**Pull request merged**